### PR TITLE
Feature logging

### PR DESCRIPTION
**Pull request by Diocruel**
Logging is now implemented as far as it will be. Most requirements have been met and the ones that are not are considered to time-consuming or unnecessary to implement for this game! 

From now one everyone should include their own logging in their new classes. Sysouts should be avoided, instead logging with appropriate log levels should be used!

Also fixed a few lingering bugs and the findbug that popped up.

<bug,feature

### CODE REVIEW

File: src/main/java/com/github/fishio/Preloader.java [hunk 1]
```diff
@@ -114,7 +119,7 @@ private static void tryPreLoad(String file, boolean pixelData) {
 		try {
 			image = new Image(file);
 		} catch (Exception ex) {
-			System.err.println("Error while trying to load image " + file);
+			log.log(LogLevel.WARNING, "Error while trying to load image: " + file);
```
Comment (ChielBruin) [inline src/main/java/com/github/fishio/Preloader.java]: Shouldn't this have the log level ERROR?
<naming

File: src/main/java/com/github/fishio/control/MainMenuController.java [hunk 2]
```diff
@@ -54,6 +58,7 @@ public void onSwitchTo() { }
 	 */
 	@FXML
 	public void startSinglePlayer() {
+		log.log(LogLevel.INFO, "Player Pressed the SingPlayer Button");
```
Comment (ChielBruin) [inline src/main/java/com/github/fishio/control/MainMenuController.java]: Typo
<naming

**Comment by Diocruel** [comment]
Chiel, I think that if fails to load the image there, the program should still be able to keep running,

File: src/main/java/com/github/fishio/control/MainMenuController.java (hunk 2 above)
Comment (Diocruel) [inline src/main/java/com/github/fishio/control/MainMenuController.java]: Chiel, I think that if fails to load the image there, the program should still be able to keep running.

<naming

File: src/main/java/com/github/fishio/Preloader.java (hunk 1 above)
Comment (Diocruel) [inline src/main/java/com/github/fishio/Preloader.java]: Chiel, I think that if fails to load the image there, the program should still be able to keep running.

<naming

Comment (ChielBruin) [inline src/main/java/com/github/fishio/Preloader.java]: You are right, WARNING is better
<naming

**Comment by ChielBruin** [comment]
Except the typo in one of the log messages, everything looks good!
<feature,naming

**Comment by ChielBruin** [comment]
wrong button..

File: src/main/java/com/github/fishio/Preloader.java (hunk 1 above)
Comment (Diocruel) [inline src/main/java/com/github/fishio/Preloader.java]: I'll change it to ERROR so it will be consistent

**Comment by Diocruel** [comment]
Implemented your suggestions @ChielBruin 

<bug,naming

**Comment by Taeir** [comment]
Looks good, good to merge!

<bug,feature

**Pull request merged**